### ISSUE TITLE
Is there a way to serialize and maintain ordering?

### ISSUE THREAD

**Issue by lambdakris** [RobThree/NUlid#13]
Is there a means of serializing an NUlid object and still maintaining it's lexicographic ordering? More specifically, is there a way or would it make sense to convert an NUlid object into a representation than can be stored and have it still maintain its order amongst other stored NUlid objects?

**Comment by RobThree** [comment]
Uh, the string representation *is* the serialized and ordered form? I'm not sure I understand the question?

**Comment by RobThree** [comment]
@lambdakris Did this answer your question? Can I close the issue? Or do you still have a question?

**Comment by lambdakris** [comment]
Hey @RobThree, took another look and looks like I may have made a mistake initially. I had written some code to test ordering, but I had failed to take into account that if I create a bunch of ulids in the same millisecond then ordering is not guaranteed unless I use the MonotonicUlidRng. This makes sense and I think I can deal with that so feel free to close the issue. However, just in case you want to double check what I was doing, here is the [example console app](https://github.com/RobThree/NUlid/files/3723058/GrokNUlid.zip) I created to verify how ordering was working.
